src/main/java/DenisCamacho/task_02/Table.java

```diff
@@ -0,0 +1,94 @@
+package main.java.DenisCamacho.task_02;
+
+public class Table {
+    private String[][] table;
+
+    public Table() {
+        table = new String[3][3];
+        cleaning();
+    }
+
+    public void insert(Player player, int row, int column) {
+        table[row][column] = String.format("%s", player.getChip());
+    }
+
+    public void cleaning() {
+        System.out.println("Cleaning the table #################################################");
```

Comment:
@denis1729 This can work as a log, but if it is part of the console application, the behavior should go to another class your Table only need to handle the table model. you can read about this in https://stackoverflow.com/questions/129921/what-is-mvc-model-view-controller